status-group-chats-spec.md

```diff
@@ -28,177 +28,105 @@ This documents describes the group chat protocol used by the status application.
 
 ## Membership updates
 
-Membership updates messages are used to propagate group chat membership changes. The transit format is described in the [Status Payload Specs](status-payload-specs.md). Here we will be describing each specific field.
-
-The format is:
+Membership updates messages are used to propagate group chat membership changes. The protobuf format is described in the [Status Payload Specs](status-payload-specs.md). Here we will be describing each specific field.
+
+The protobuf messages are:
+
+```protobuf
+// MembershipUpdateMessage is a message used to propagate information
+// about group membership changes.
+message MembershipUpdateMessage {
+  // The chat id of the private group chat
```

Comment:
Is this unique to private group chats? vs "group chat" as a construct (I know we don't - currently, anymore - support public group chats).

Reply:
Yes, that's only used for private group chats